uiowa/uiowa
# Do not allow configuring page header section.

Resolves #2694

For non-admins, this removes the ability to configure the header section if it exists. This goes for new nodes, default content going forward and existing default content out there that didn't have this lock in place.

**The Gotchas:**
- Existing sites that customized these regions will need an Admin to fix them.
- Any site thinking they should be able to edit these, will need to be split.

# To Test

As a webmaster of a v3 website, you should not be able to modify header section configuration for

- New pages
- about page of a fresh install
- node/1 if the defaults are reset and a header is provided
- existing node/6 (about) pages
- make sure v2 sites don't break as a result of the update hook
- same goes for admissions
